eng/bomgenerator/src/main/java/com/azure/tools/bomgenerator/Main.java

```diff
@@ -3,86 +3,74 @@
 
 package com.azure.tools.bomgenerator;
 
+import java.io.FileNotFoundException;
 import java.util.regex.Matcher;
 import static com.azure.tools.bomgenerator.Utils.ANALYZE_MODE;
+import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_INPUTDIRECTORY;
+import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_OUTPUTDIRECTORY;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_OVERRIDDEN_INPUTDEPENDENCIES_FILE;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_INPUTFILE;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_MODE;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_OUTPUTFILE;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_POMFILE;
+import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_REGEX;
 import static com.azure.tools.bomgenerator.Utils.COMMANDLINE_REPORTFILE;
+import static com.azure.tools.bomgenerator.Utils.EMPTY_STRING;
 import static com.azure.tools.bomgenerator.Utils.GENERATE_MODE;
 import static com.azure.tools.bomgenerator.Utils.validateNotNullOrEmpty;
 import static com.azure.tools.bomgenerator.Utils.validateValues;
 
 public class Main {
 
     public static void main(String[] args) {
-        BomGenerator generator = new BomGenerator();
-        parseCommandLine(args, generator);
-        generator.run();
+        BomGenerator generator = null;
+        try {
+            generator = parseCommandLine(args);
+        } catch (FileNotFoundException e) {
+            System.out.println("Error occurred.");
+            e.printStackTrace();
+        }
+        if(!generator.run()) {
```

Comment:
Do we want to run the generator if an error occurred when parsing the arguments?